src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -161,6 +396,75 @@ can also be used to make that distinction. At what level of detail to make the
 distinction (e.g. just between RARE and FLASH, or between RARE, FLASH, and
 FLASHsubsampled) remains at the discretion of the researcher.
 
+#### The `echo` entity 
```

Comment:
Does not specify any units in this section.

Reply:
@satra should not units be specified in the `EchoTime` entry of the `Timing Parameters`  metadata table? On the other hand, I totally agree with your [comment](https://github.com/bids-standard/bids-specification/pull/508#issuecomment-661163395) regarding data types, happy to adapt BEP001 once there is a solution.

Reply:
@agahkarakuzu - you are correct the units are in the timing table. perhaps what's needed is a link between the mention in this section and the link to the table. i know this is being discussed in a separate issue, so for this BEP, please feel free to resolve this issue for now.